src/Digdir.Domain.Dialogporten.Application/Features/V1/ServiceOwner/Dialogs/Queries/Search/SearchDialogQuery.cs

```diff
@@ -89,10 +90,8 @@ public async Task<SearchDialogResult> Handle(SearchDialogQuery request, Cancella
             .WhereIf(request.VisibleAfter.HasValue, x => request.VisibleAfter <= x.VisibleFrom)
             .WhereIf(request.VisibleBefore.HasValue, x => x.VisibleFrom <= request.VisibleBefore)
             .WhereIf(request.Search is not null, x =>
-                x.Title!.Localizations.AsQueryable().Any(searchExpression) ||
-                x.Body!.Localizations.AsQueryable().Any(searchExpression) ||
-                x.SearchTags.Any(x => x.Value.Equals(request.Search, StringComparison.OrdinalIgnoreCase)) ||
-                x.SenderName!.Localizations.AsQueryable().Any(searchExpression)
+                x.Content.Any(x => x.Value.Localizations.AsQueryable().Any(searchExpression)) ||
+                x.SearchTags.Any(x => EF.Functions.ILike(x.Value, request.Search!))
```

Comment:
Egentlig off-topic for denne PR-en, men vi må se på dette etterhvert. Pt. hvis en har "foobar" som search tag og søker på "foobar baz" får man ingen treff, og det er ikke helt gunstig.